aqua/entrypoint_mediawiki.sh

```diff
@@ -3,7 +3,7 @@
 MW_DIR=/var/www/html
 if [ -f /backup/LocalSettings.php ]; then
     if [ ! -L $MW_DIR/LocalSettings.php ]; then
-        ln -s /backup/LocalSettings.php $MW_DIR/LocalSettings.php
+       echo -e '<?php\nrequire_once "/backup/LocalSettings.php";' > $MW_DIR/LocalSettings.php
```

Comment:
@it-spiderman Thank you, this would have taken us quite some time to figure out.